custom/icds_reports/tests/test_nic_indicators_api.py

```diff
@@ -1,27 +1,52 @@
 
-from datetime import date
+from datetime import datetime, date
 from django.test import TestCase
 from custom.icds_reports.utils.data_accessor import get_inc_indicator_api_data
+from mock import mock
+
+
+class SecondDayOfMay(datetime):
+    @classmethod
+    def utcnow(cls):
+        return datetime(2017, 5, 2)
```

Comment:
you may be able to use `freezegun` to do a similar thing in a more consistent way. It should change all datetime functions instead of just the ones that are explicitly mocked